application/src/main/resources/application.yml

```diff
@@ -24,6 +24,13 @@ management:
   endpoint:
     health:
       enabled: true
+      probes:
+        enabled: true
```

Comment:
As I understand it, these probes are enabled by default according to (https://www.baeldung.com/spring-liveness-readiness-probes) when using a Kubernetes environment. Why do we enable it for all environments?

Reply:
I don't trust automatic detection, "automatic detection" usually doesn't work 100% of the time. Also, i think it's nice to have alive/ready split up.